### ISSUE TITLE
loss function 

### ISSUE THREAD

**Issue by lww28** [chenlin9/high_order_dsm#1]
Hey,

Thank you for your incredible work! I have a question regards the loss functions in the training and testing phase. In the training phase, the loss function for the dsm divided by sigma**2, while it is not done in the testing phase as shown below. Moreover, why do we need to divided by sigma**2, the original dsm doesn't have this term. Could you please help me with this? Thank you so much!!

For example, in low_rank_hybrid_covariance_runner.py
In the training phase:
loss1 = 10. * self.loss_op1(s1, X, self.sigma) / (self.sigma ** 2)  # dsm training s1
loss2 = self.loss_op2(s1, model, X, self.sigma) # second order sm loss
loss = loss1 + loss2
......
In the testing phase:
            with torch.no_grad():
                for batch_idx, (X, _) in enumerate(test_loader):
                    X = X.cuda(non_blocking=True)
                    X = X.view(X.shape[0], -1)
                    loss1 = self.loss_op1(s1, X, self.sigma) # here 
                    loss2 = self.loss_op2(s1, model, X, self.sigma)
